ruby/puma/README.md

```diff
@@ -0,0 +1,16 @@
+# Ruby sample app using Puma web server
+
+## Building
+
+`pack build puma-sample --buildpack gcr.io/paketo-community/ruby`
+
+## Running
+
+`docker run --interactive --tty --publish 9292:8080 puma-sample`
```

Comment:
```suggestion
`docker run --interactive --tty --publish 9292:9292 puma-sample`
```

Note that this won't work because the buildpack doesn't currently support custom port binding (https://github.com/paketo-buildpacks/puma/issues/7). Probably better to just use port 9292 or alternatively modify rackup config.ru file to set the port to 8080